app/application/airflow/dags/send_msg_2_dag.py

```diff
@@ -0,0 +1,41 @@
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+from logic.send_random_message import (
```

Comment:
no empty line

Reply:
Ruff tuned, known first and third party added to isort for auto grouping by imports.